src/style.css

```diff
@@ -0,0 +1,2 @@
+@import "./styles/card.css";
+@import "./styles/stars.css";
```

Comment:
i don't see these files in your pr, add them with `git add <file_name>` command, then commit and push them